react/week-2/lesson.md

```diff
@@ -443,6 +443,14 @@ function FruitCounter() {
 }
 ```
 
+
+| **Exercise C** |
```

Comment:
Could we move this above "Where does State live"/"Setting multiple states"? Those 2 sections aren't required to complete the exercise and are really more of an FYI.

Reply:
Exercise split in half - half is before 'Where does State live', half is after.

Reply:
I like the exercise, thanks (I was struggling to come up with a simple realistic that needed 2 states without replacing the counter entirely).